v2/datastream-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/DataStreamToSpannerSimpleIT.java

```diff
@@ -187,6 +190,165 @@ public void migrationTestWithInsertsOnly() {
     assertMovieTableContents();
   }
 
+  @Test
+  public void migrationTestWithAllDatatypeConversions() {
```

Comment:
Both this and the below function name sounds similar. Can you rename the function properly? Maybe similar to `allDatatypeDefaultMapping` and `allDatatypeConversionMapping` ?